fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -578,6 +893,66 @@ struct RangeFileWriter {
 	Key lastValue;
 };
 
+SnapshotFileBackupCipherKeysCtx getCipherKeyCtx(Arena& arena) {
```

Comment:
U might need to support two lookup modes:
1. GetLatestCipherKey - {domainId} based lookup - for writing new files
2. GetCipherKey - {domainId, baseCipherId} based lookup to enable decryption of already written files. 

Also, I think lookup needs to integrate with BlobCipherKeyCache to providing in-memory caching semantics, the cache supports `BlobCipherKey` format, so no transformation needed I think

Reply:
Good point. I'll keep this in mind. I think we can ignore it for this pr and incorporate it when we bring in the EKP